flang/lib/Lower/OpenMP/ClauseProcessor.cpp

```diff
@@ -842,8 +842,8 @@ bool ClauseProcessor::processLink(
 mlir::omp::MapInfoOp
 createMapInfoOp(fir::FirOpBuilder &builder, mlir::Location loc,
                 mlir::Value baseAddr, mlir::Value varPtrPtr, std::string name,
-                mlir::SmallVector<mlir::Value> bounds,
-                mlir::SmallVector<mlir::Value> members, uint64_t mapType,
+                mlir::ArrayRef<mlir::Value> bounds,
+                mlir::ArrayRef<mlir::Value> members, uint64_t mapType,
```

Comment:
Could you change this to `llvm::`?  The `llvm` namepace is used everywhere else in this file for these containers.

Reply:
Done. Thanks for noticing.